libsolidity/analysis/FunctionCallGraph.cpp

```diff
@@ -56,7 +56,13 @@ CallGraph FunctionCallGraphBuilder::buildCreationGraph(ContractDefinition const&
 		// Functions called from the inheritance specifier should have an edge from the constructor
 		// for consistency with functions called from constructor modifiers.
 		for (auto const& inheritanceSpecifier: base->baseContracts())
+		{
+			if (auto const baseContract = dynamic_cast<ContractDefinition const*>(inheritanceSpecifier->name().annotation().referencedDeclaration))
+				if (baseContract != &_contract)
+					builder.m_graph.bytecodeDependency.emplace(baseContract, inheritanceSpecifier.get());
```

Comment:
So do we need inherited contracts in `bytecodeDependency` after all? I think that the conclusion from https://github.com/ethereum/solidity/pull/10228#discussion_r594415269 was that we don't.

Or is it needed because of the case with unreachable functions?

Reply:
A small test showed that only the ASTJSON output is changed (as it has now less dependencies), all other tests seem to continue to work. So I suppose we could remove it.